vantage/aws_provider_resource.go

```diff
@@ -62,27 +63,19 @@ func (r AwsProviderResource) Create(ctx context.Context, req resource.CreateRequ
 		return
 	}
 
-	// Convert from Terraform data model into API data model
-	createReq := AwsProviderResourceAPIModel{
-		CrossAccountARN: data.CrossAccountARN.ValueString(),
-		BucketARN:       data.BucketARN.ValueString(),
+	params := integrationsv1.NewCreateIntegrationsAWSParams()
+	model := &modelsv1.PostIntegrationsAws{
+		CrossAccountArn: data.CrossAccountARN.ValueStringPointer(),
```

Comment:
`cross_account_arn` is marked as `requires` in the api, so it's exposed as a pointer. If the pointer is nil then it knows the value was not set. If it was just a string, then it'd be ambiguous if the empty string value `""` was because the user was specifying empty string or because they didn't set it.